Euphorolog.Database/Models/Stories.cs

```diff
@@ -10,12 +10,10 @@ namespace Euphorolog.Database.Models
 {
     public class Stories
     {
-            [Key,Required]
             public string storyId { get; set; }
             public int storyNo { get; set; }
```

Comment:
Property names use Pascal Casing convention. For example, `StoryId`, `StoryTitle`